### PR TITLE
New breaking api, compile() is an alias to compileStandardWrapper()

### PR DESCRIPTION
**Pull request by axic**
Implements #120 and https://github.com/ethereum/solidity/issues/3452

This is the hard-breaking option, while #122 implements a more incremental change, but carries a lot more baggage.

Depends on #256. Needs test changes.

### CODE REVIEW

**Comment by leonardoalt** [comment]
Rebased

**Comment by axic** [comment]
Rebased. This is the last solc-js change for 0.5.0.

**Comment by chriseth** [comment]
I think we should get this merged even without tests because otherwise it stalls another PR on the solidity repo. Also I would say it is fine for tests to temporarily fail on this repo if it allows tests to be continuously green on the solidity repo.

**Comment by chriseth** [comment]
Note that the tests of this PR have to be run against the soljson.js coming out of the still unmerged PR on the solidity side. Locally running only resulted in one test failure related to the package version. Because of that, I will merge this now.

File: wrapper.js [hunk 1]
```diff
@@ -48,25 +48,25 @@ function setupMethods (soljson) {
...
+  if ('x_compileJSONCallback' in soljson) {
     var compileInternal = soljson.cwrap('compileJSONCallback', 'string', ['string', 'number', 'number']);
     compileJSONCallback = function (input, optimize, readCallback) {
       return runWithReadCallback(readCallback, compileInternal, [ input, optimize ]);
     };
   }
 
   var compileStandard = null;
-  if ('_compileStandard' in soljson) {
+  if ('x_compileStandard' in soljson) {
```
Comment (axic) [inline wrapper.js]: This is broken. It was only there for testing to be triggered.

**Pull request merged**